### PR TITLE
Working groups

### PR DESCRIPTION
**Pull request by jamiecobbett**
## Here is the "short" example:
short-before
short-after

<hr>

## "with_policies" example
with_policies-before
with_policies-after

<hr>

## "long" example
long-before
long-after

### CODE REVIEW

**Comment by fofr** [comment]
Can you add some screenshots of before and after?

File: test/test_helper.rb [hunk 1]
```diff
@@ -52,9 +59,13 @@ def assert_has_component_breadcrumbs(breadcrumbs)
...
 
   def assert_has_contents_list(contents)
+    assert page.has_css?(".dash-list"), "Failed to find an element with a class of dash-list"
     within ".dash-list" do
       contents.each do |heading|
-        assert page.has_css?("a[href=\"##{heading[:id]}\"]", text: heading[:text])
+        selector = "a[href=\"##{heading[:id]}\"]"
+        text = heading.fetch(:text)
+        assert page.has_css?(selector), "Failed to find an element matching: #{selector}"
+        assert page.has_css?(selector, text: text), "Failed to find an element matching #{selector} with text: #{text}"
```
Comment (fofr) [inline test/test_helper.rb]: :+1: :100:

File: test/test_helper.rb [hunk 2]
```diff
@@ -39,12 +39,19 @@ def assert_has_component_title(title)
     end
   end
 
-  def assert_has_component_govspeak(content)
-    within shared_component_selector("govspeak") do
+  def assert_has_component_govspeak(content, index: 1)
```
Comment (fofr) [inline test/test_helper.rb]: Nice pattern. I think we could re-use this pattern on one of the stats announcements which has multiple components of the same type, metadata I think. cc @dsingleton

**Comment by fofr** [comment]
Looks good to me – pending a visual diff with screenshots, I like that you didn't need to write any new styles :+1:

**Comment by jamiecobbett** [comment]
@fofr screenshots added

File: test/test_helper.rb (hunk 2 above)
Comment (dsingleton) [inline test/test_helper.rb]: That _is_ nice.

File: app/views/content_items/working_group.html.erb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    <%= render 'govuk_component/govspeak',
+        content: @content_item.body,
+        direction: page_text_direction %>
+
+    <% @additional_body = capture do %>
+      <% if @content_item.policies.any? %>
+        <h2 id="policies">Policies</h2>
+        <ul>
+          <% @content_item.policies.each do |policy| %>
+            <%= content_tag(:li) do %>
```
Comment (fofr) [inline app/views/content_items/working_group.html.erb]: This could be `<li>`

**Pull request merged**